packages/strapi/lib/Strapi.js

```diff
@@ -66,6 +67,8 @@ class Strapi {
     this.eventHub = createEventHub();
 
     this.requireProjectBootstrap();
+
+    createNotifyUpdater(this).notify();
```

Comment:
would make more sense if it was `createUpdateNotifier`

Reply:
totally :p